mne/decoding/tests/test_base.py

```diff
@@ -0,0 +1,126 @@
+# Author: Jean-Remi King, <jeanremi.king@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from numpy.testing import assert_array_equal, assert_array_almost_equal
+from nose.tools import assert_true, assert_equal, assert_raises
+from ...utils import requires_sklearn
+from ..base import _get_inverse_funcs, LinearModel, get_coef
+from ..search_light import _SearchLight
+
+
+@requires_sklearn
+def test_get_coef():
+    from sklearn.base import TransformerMixin, BaseEstimator
+    from sklearn.pipeline import make_pipeline
+    from sklearn.preprocessing import StandardScaler
+    from sklearn.linear_model import LinearRegression
+
+    # Define a classifier, an invertible transformer and an non-invertible one.
+
+    class Clf(BaseEstimator):
+        def fit(self, X, y):
+            return self
+
+    class NoInv(TransformerMixin):
+        def fit(self, X, y):
+            return self
+
+        def transform(self, X):
+            return X
+
+    class Inv(NoInv):
+        def inverse_transform(self, X):
+            return X
+
+    np.random.seed(0)
```

Comment:
the `rng = np.random.RandomState(0)` pattern is generally preferred but okay to leave as is